SPECS/kernel-rt/config

```diff
@@ -5721,7 +5721,20 @@ CONFIG_MXM_WMI=m
 # CONFIG_ASUS_WMI is not set
 # CONFIG_MERAKI_MX100 is not set
 # CONFIG_EEEPC_LAPTOP is not set
-# CONFIG_X86_PLATFORM_DRIVERS_DELL is not set
```

Comment:
Signature file needs to be updated for this package

Reply:
Fixed